envConfig/live.env

```diff
@@ -11,9 +11,11 @@ LOG_DIR='/var/log/simorgh'
 LOG_LEVEL='info'
 LOG_TO_CONSOLE=false
 SIMORGH_APP_ENV=live
-SIMORGH_CONFIG_CACHE_ITEMS=100
+SIMORGH_CONFIG_CACHE_ITEMS=200
```

Comment:
Why has this value been changed? We will be caching no more items with this chnage to my knowledge?